### PR TITLE
core: set invoice id by iterating chunks in order to avoid lock timeouts

### PR DESCRIPTION
**Pull request by mmadariaga**
Long queries don't play well with percona cluster

#### Type Of Change 
- [X] Small bug fix
- [ ] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

File: library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/BillableCallDoctrineRepository.php [hunk 1]
```diff
@@ -279,21 +279,29 @@ function (array $row) {
...
-            ->createQueryBuilder('self')
-            ->update($this->_entityName, 'self')
-            ->set('self.invoice', ':invoiceId')
-            ->setParameter(':invoiceId', $invoice->getId())
-            ->addCriteria(
-                CriteriaHelper::fromArray([
-                    ['id', 'in', $targetIds],
-                ])
+
+        $tagetIdChunks = array_chunk($targetIds, 2500);
```
Comment (cruzccl) [inline library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/BillableCallDoctrineRepository.php:283]: targetIdChunks (just a typo)

File: library/Ivoz/Provider/Domain/Service/Invoice/Generator.php [hunk 2]
```diff
@@ -188,8 +188,6 @@ protected function _getCallData(InvoiceInterface $invoice): array
         $invoiceTz = new \DateTimeZone(
             $company->getDefaultTimezone()->getTz()
         );
-        $utcTz = new \DateTimeZone('UTC');
```
Comment (cruzccl) [inline library/Ivoz/Provider/Domain/Service/Invoice/Generator.php:191]: Why? :thinking:

Comment (mmadariaga) [reply library/Ivoz/Provider/Domain/Service/Invoice/Generator.php:191]: Unused variable

File: library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/BillableCallDoctrineRepository.php (hunk 1 above)
Comment (mmadariaga) [reply library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/BillableCallDoctrineRepository.php]: fixed, thanks!

**Review by cruzccl — APPROVED** [review]

**Pull request merged**